plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -88,6 +89,8 @@ private CloudSdk getCloudSdk() {
     CloudSdk cloudSdk = new CloudSdk.Builder()
                           .addStdOutLineListener(new MessageConsoleWriterOutputLineListener(outputStream))
                           .addStdErrLineListener(new MessageConsoleWriterOutputLineListener(outputStream))
+                          .appCommandMetricsEnvironment(CloudToolsInfo.METRICS_NAME)
```

Comment:
For now we have only this location, but I can imagine a helper/factory that always sets these two properties so that we don't need to repeat, and we won't forget to do so in future instances.
We can work on that when it comes to implementing a feature requiring CloudSdk.Builder again.